core/idl/wallet/cosmos/wallet.djinni

```diff
@@ -99,6 +99,16 @@ CosmosLikeAccount = interface +c {
         getLatestValidatorSet(callback: ListCallback<CosmosLikeValidator>);
         # Get information about one validator
         getValidatorInfo(validatorAddress: string, callback: callback2<optional<CosmosLikeValidator>>);
+        # Get Total balance of account. Sum of spendable, delegated, pending rewards, and pending unbondings
+        getTotalBalance(callback: Callback<Amount>);
+        # Get Total amount in delegation of account.
+        getDelegatedBalance(callback: Callback<Amount>);
+        # Get Total pending rewards of account.
+        getPendingRewards(callback: Callback<Amount>);
```

Comment:
Suggested name: `getDelegatedBalanceAmount` ?
For compatibility with https://github.com/gagbo/lib-ledger-core/pull/24

Reply:
I'll rename it `PendingRewardsBalance` to mark it as a function to get the total balance of the account. I'll probably change the unbonding function name too